src/main/java/com/rackspace/ceres/app/services/DataTablesStatements.java

```diff
@@ -69,29 +73,22 @@ String partitionTableNameSuffix(Duration width) {
     return "p_" + width.toString().toLowerCase();
   }
 
-  private void buildRawStatements(AppProperties appProperties) {
-    rawInsert = "INSERT INTO " + tableNameRaw(appProperties.getRawPartitionWidth())
-        + " (" +
-        String.join(",", TENANT, TIME_PARTITION_SLOT, SERIES_SET_HASH, TIMESTAMP, VALUE)
-        + ")"
-        + " VALUES (?, ?, ?, ?, ?)";
-    rawQuery = "SELECT " + String.join(",", TIMESTAMP, VALUE)
-        + " FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
-        + " WHERE " + TENANT + " = ?"
-        + "  AND " + TIME_PARTITION_SLOT + " = ?"
-        + "  AND " + SERIES_SET_HASH + " = ?"
-        + "  AND " + TIMESTAMP + " >= ? AND " + TIMESTAMP + " < ?";
-    rawDelete = "DELETE FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
-        + " WHERE " + TENANT + " = ?"
-        + "  AND " + TIME_PARTITION_SLOT + " = ?";
-    rawDeleteWithSeriesSetHash = "DELETE FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
-        + " WHERE " + TENANT + " = ?"
-        + "  AND " + TIME_PARTITION_SLOT + " = ?"
-        + " AND series_set_hash = ?";
-    rawGetHashQuery =
-        "SELECT series_set_hash FROM " + tableNameRaw(appProperties.getRawPartitionWidth())
-            + " WHERE " + TENANT + " = ?"
-            + "  AND " + TIME_PARTITION_SLOT + " = ?";
+  private void buildRawStatements(AppProperties appProperties) throws IOException {
+    rawInsert = String
+        .format(SpringResourceUtils.readContent("cql-queries/raw_insert_query.cql"),
+            tableNameRaw(appProperties.getRawPartitionWidth()), String.join(",", TENANT, TIME_PARTITION_SLOT, SERIES_SET_HASH, TIMESTAMP, VALUE));
+    rawQuery = String
+        .format(SpringResourceUtils.readContent("cql-queries/raw_select_query.cql"),
+            String.join(",", TIMESTAMP, VALUE), tableNameRaw(appProperties.getRawPartitionWidth()));
+    rawDelete = String
+        .format(SpringResourceUtils.readContent("cql-queries/raw_delete_query.cql"),
+            tableNameRaw(appProperties.getRawPartitionWidth()));
+    rawDeleteWithSeriesSetHash = String
+        .format(SpringResourceUtils.readContent("cql-queries/raw_delete_with_series_set_hash.cql"),
+            tableNameRaw(appProperties.getRawPartitionWidth()));
+    rawGetSeriesSetHashQuery = String
+        .format(SpringResourceUtils.readContent("cql-queries/raw_select_series_set_hash.cql"),
```

Comment:
So this is not what my expectation was. this will get loaded from the resource file every time its referenced. The value should be stored in a class level final string similar to this https://github.com/racker/salus-telemetry-monitor-management/blob/master/src/main/java/com/rackspace/salus/monitor_management/services/MonitorManagement.java#L199-L200

Reply:
@harageth , for now, these are initialized at bean creation time through the constructor code, so it won't be loaded from the resource file every time while referencing but yes, it's not stored in a class level variable.

one more thing, should I go ahead with putting cql queries directly in class level final string instead of reading it from template files as these are simple query.

Reply:
Ahhhh I see that now. If doing a template like this then you want to write the template as complete as possible. So if each specific query is for a specific table name then the query should just have the schema name written in it.

That being said I would be fine with these just being turned into final strings without having templates.

Reply:
as these all cql query are quite simple and straight forward, making it final string would be make more sense.